cypress/integration/ete/consent_token/consent_token.2.cy.ts

```diff
@@ -0,0 +1,48 @@
+describe('Consent token', () => {
+  it('Consent token flow with valid token', () => {
+    cy.createTestUser({
+      isUserEmailValidated: true,
+    }).then(({ emailAddress, finalPassword }) => {
+      // Sign our new user in
+      cy.visit(
+        `/signin?returnUrl=${encodeURIComponent(
+          `https://${Cypress.env('BASE_URI')}/consents`,
+        )}`,
+      );
+      cy.get('input[name=email]').type(emailAddress);
+      cy.get('input[name=password]').type(finalPassword);
+      cy.get('[data-cy="main-form-submit-button"]').click();
+      cy.url().should('include', '/consents');
```

Comment:
this can become 
```suggestion
    cy.createTestUser({
      isUserEmailValidated: true,
    }).then(({ emailAddress, cookies }) => {
      // SC_GU_U is required for the cy.getTestUserDetails
      const scGuU = cookies.find((cookie) => cookie.key === 'SC_GU_U');
      if (!scGuU) throw new Error('SC_GU_U cookie not found');
      cy.setCookie('SC_GU_U', scGuU?.value);
```